.github/workflows/dependencies.yml

```diff
@@ -34,6 +36,7 @@ jobs:
           git checkout $GITHUB_SHA
 
       - name: Commit this bump
+        # Is it mandatory to create a new commit? Or does it exist as a way to communicate why the version was updated.
```

Comment:
Did you forget to remove this comment?

Reply:
I did not. This comment serves as a reminder that no one is completely certain how this exactly functions. If somebody does know how it does, they should document it as this is a quite obscure way of handling dependency updates.

Reply:
I don't see what value this comment adds.

Reply:
I don't know about the other projects but for Nimbus, the `Checkout this ref` step update `nim-libp2p` - which is a submodule dependency. This step is committing this update.

Reply:
I'm aware of how it functions. I'm simply not aware of its true purpose and/or whether that's the only way to handle it.

I've asked Nimbus and they confirmed they're still using this as a mechanism to test integrations of every libp2p trunk commit with nimbus-eth2.

Knowing this, there's no need for this comment anymore.